evalml/tests/objective_tests/test_standard_metrics.py

```diff
@@ -662,9 +662,8 @@ def test_mse_linear_model():
 def test_mcc_catches_warnings():
     y_true = [1, 0, 1, 1]
     y_predicted = [0, 0, 0, 0]
-    with pytest.warns(RuntimeWarning) as record:
```

Comment:
Between [0.24.0](https://github.com/scikit-learn/scikit-learn/blob/2beed55847ee70d363bdbfe14ee4401438fba057/sklearn/metrics/_classification.py#L875) and [1.0.0+](https://github.com/scikit-learn/scikit-learn/blob/0d378913be6d7e485b792ea36e9268be31ed52d0/sklearn/metrics/_classification.py#L906), the implementation was changed to just return a matthews_corrcoef score of 0.0 if the denominator was 0 (previously caused invalid divide by 0)